scheduler/scheduler_test.go

```diff
@@ -71,6 +72,14 @@ func TestScheduler(t *testing.T) {
 		Convey("createEremeticScheduler", func() {
 			s := createEremeticScheduler()
 			So(s.tasksCreated, ShouldEqual, 0)
+			So(cap(s.tasks), ShouldEqual, 100)
+		})
+
+		Convey("createEremeticScheduler with configured channel size", func() {
+			MaxQueueSize = 200
```

Comment:
When making this runtime configurable I don't think it makes sense to keep this as a global variable. lets add a paramater to `createEremeticScheduler`?

I've seen this pattern, using a options struct, a few times and I quite like it
```
createEremeticScheduler(SchedulerOptions{
    QueueSize: 200
})
```